woocommerce/woocommerce-ios
# [CPP Onboarding] Implement view for plugin not installed

Part of #4611 

This adds a specific onboarding error screen for the case when the store is in a supported country but the WooCommerce Payments plugin is not installed.

## To test

You should have a store with an address in the US and without the WooCommerce Payments installed

1. Go to Settings > In-Person Payments. The app should show a screen prompting you to install the extension.
2. From wp-admin, install WooCommerce Payments from WooCommerce > Extensions > Payments.
3. Tap on Refresh After Installing.
4. The app should show a generic error (since payments aren't fully set up yet).

Extension not installed | After tapping Refresh
-|-
![Screen Shot 2021-08-06 at 11 50 16](https://user-images.githubusercontent.com/8739/128492534-08822040-00ba-4de0-bbc8-8ca4ce69ef5c.png)|![Screen Shot 2021-08-06 at 11 11 42](https://user-images.githubusercontent.com/8739/128487773-26076b02-dd96-4f49-8835-85895fe5b4d6.png)


Update release notes:
